django/contrib/admin/options.py

```diff
@@ -555,7 +556,12 @@ def wrapper(*args, **kwargs):
             url(r'^add/$', wrap(self.add_view), name='%s_%s_add' % info),
             url(r'^(.+)/history/$', wrap(self.history_view), name='%s_%s_history' % info),
             url(r'^(.+)/delete/$', wrap(self.delete_view), name='%s_%s_delete' % info),
-            url(r'^(.+)/$', wrap(self.change_view), name='%s_%s_change' % info),
+            url(r'^(.+)/edit/$', wrap(self.change_view), name='%s_%s_change' % info),
```

Comment:
Is there a reason `edit` was chosen instead of `change`. It's the only pattern that differs from it's name.